packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -819,17 +801,18 @@ mod default {
         INSTALL_BUILTIN.call_once(|| {
             INSTALL_BUILTIN_RUNNING.store(true, Ordering::Release);
 
-            Report::install_debug_hook(location);
+            Report::install_debug_hook::<Location>(location);
 
             #[cfg(rust_1_65)]
-            Report::install_debug_hook(backtrace);
+            Report::install_debug_hook::<Backtrace>(backtrace);
 
             #[cfg(feature = "spantrace")]
-            Report::install_debug_hook(span_trace);
+            Report::install_debug_hook::<SpanTrace>(span_trace);
```

Comment:
I added these to ensure it's actually calling the right hook. Before, I accidentally created a hook for `&Location` instead of `Location`